src/cli/README.md

```diff
@@ -98,6 +99,132 @@ Done
 
 ## OpenThread Command Details
 
+### bbr
+
+Show current Primary Backbone Router information for Thread 1.2 device.
+
+```bash
+> bbr
+BBR Primary:
+server16: 0xE400
+seqno:    10
+delay:    120 secs
+timeout:  300 secs
+Done
+```
+
+```bash
+> bbr
+BBR Primary: None
+Done
+```
+
+### bbr state
+Show local Backbone state ([`Disabled`,`Primary`, `Secondary`]) for Thread 1.2 FTD.
+
+`OPENTHREAD_CONFIG_BACKBONE_ROUTER_ENABLE` is required.
+
+
+```bash
+> bbr state
+Disabled
+Done
+
+> bbr state
+Primary
+Done
+
+> bbr state
+Secondary
+Done
+```
+
+### bbr enable
+Enable Backbone Router Service for Thread 1.2 FTD.
+`SRV_DATA.ntf` would be triggerred for attached device if there is no
+Backbone Router Service in Thread Network Data. 
+
+`OPENTHREAD_CONFIG_BACKBONE_ROUTER_ENABLE` is required.
+
+```bash
+> bbr enable
+Done
+```
+
+### bbr disable
+Disable Backbone Router Service for Thread 1.2 FTD.
+`SRV_DATA.ntf` would be triggerred if Backbone Router is Primary state.
+o
+`OPENTHREAD_CONFIG_BACKBONE_ROUTER_ENABLE` is required.
+
+```bash
+> bbr disable
+Done
+```
+
+### bbr register
+Register Backbone Router Service for Thread 1.2 FTD.
+`SRV_DATA.ntf` would be triggerred for attached device.
+
+`OPENTHREAD_CONFIG_BACKBONE_ROUTER_ENABLE` is required.
+
+```bash
+> bbr register
+Done
+```
+
+### bbr local
+
+Show local Backbone Router configuration for Thread 1.2 FTD.
+
+`OPENTHREAD_CONFIG_BACKBONE_ROUTER_ENABLE` is required.
+
+```bash
+> bbr local
+BBR local:
+seqno:    10
+delay:    120 secs
+timeout:  300 secs
+Done
+```
+
+### bbr local \[seqno \<seqno\>\] \[delay \<delay\>\] \[timeout \<timeout\>\]
```

Comment:
> for your suggestion, how could it avoid the check on whether or not it is SBBR and have advantages in guarantee one SRV_DATA.ntf?

The check will be done by the `bbr local reg` command handler, so the CLI user does not need to check it.

Reply:
Maybe I didn't get your point. Could you please explain a bit more on the usage of your proposal? 
now I know you want to check whether or not its SBBR in CLI app. One question is when the user need to add `reg`?

Reply:
This command is almost same as `setBbrDataset` in THCI. Same as we have proposed to add a `forceSendSvrDataNtf` argument ot `setBbrDataset`, the `reg` argument for `bbr local` command is for the same purpose.

Reply:
I think the conclusion in that JIRA is `setBbrDataset()` without `forceSendSvrDataNtf` and use fake SRV_DATA.ntf as it could meet existing TPs though it has some assumptions.

Reply:
Even if there are two SRV_DATA.ntf, what's the impact? why is it so important to keep 1 SRV_DATA.ntf?

I think in general user scenarios, OpenThread (CLI or API) Users only need to config Bbr Dataset and leave other things to OpenThread Core. User should be aware of the impact of explicitly issue `bbr register` or call `otBackboneRouterRegisterLocal()`

Reply:
The possible impact of the test case you concerns is that for SBBR, `setBbrDataset()` may only update local Bbr Dataset but not add it to local server data, when the fake SRV_DATA.ntf occurs, original PBBR will deregister due to lower sequence number. And SBBR would also `deregister` due to network data inconsistency as its own bbr dataset is not in local server dataset yet. So there might be one interim period that there will be no PBBR in the network. But afterwards, both SBBR and original PBBR will try to register and finally SBBR will become new PBBR due to higher sequence number, which should also be the expecting situations of TP/TH. 
Since it is decided to use the constructed fake SRV_DATA.ntf, I think Test Harness should be robust for such SRV_DATA.ntf message exchanges.
Thoughts?

Reply:
I am fine with the current `bbr config` without `reg` argument since it is not essential anyway.